test/instrumentation/stability-utils.sh

```diff
@@ -26,13 +26,7 @@ stability_check_setup() {
   kube::golang::verify_go_version
   kube::util::ensure-temp-dir
   cd "${KUBE_ROOT}"
-  export KUBE_EXTRA_GOPATH=$KUBE_TEMP
   kube::golang::setup_env
-  pushd "${KUBE_EXTRA_GOPATH}" >/dev/null
-    touch go.mod
-    GO111MODULE=on go mod edit -module=example.com/mod
-    GO111MODULE=on go get "gopkg.in/yaml.v2"
-  popd >/dev/null
```

Comment:
do we know why we were doing this block ??

Reply:
I spent some time last night looking into `git log`, `git blame`, and various PRs, and could not figure why it was done (initially in commit 73d797d474190bc3e0, PR #98739). My guess is gopkg.in/yaml.v2 was not vendored, or that a bad/old version was vendored. Perhaps @logicalhan can shed some light as to why?

Currently, yaml.v2 v2.4.0 is vendored, and the `go get` being removed is downloading the same version, meaning this code is basically a no-op.